terraform/20-app/ecs.service.front-end.tf

```diff
@@ -65,6 +65,14 @@ module "ecs_service_front_end" {
         {
           name  = "RUM_APPLICATION_ID"
           value = module.cloudwatch_rum_front_end.rum_application_id
+        },
+        {
+          name  = "REDIS_HOST"
+          value = "rediss://${aws_elasticache_serverless_cache.front_end_elasticache.endpoint.0.address}:${aws_elasticache_serverless_cache.front_end_elasticache.endpoint.0.port}"
+        },
+        {
+          name  = "AUTH_ENABLED",
```

Comment:
The frontend can use this env var to decide whether to leverage Next.js's built-in caching solution or keep as is and rely on the CDN